_posts/2020-06-01-Fortran-Newsletter-June-2020.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: "Fortran newsletter: June 2020"
+category: newsletter
+author: Milan Curcic
+---
+
+```fortran
+print *, 'Hello, World!'
+```
+
+Welcome to this monthly Fortran newsletter.
+It will come out on the first calendar day of every month,
+detailing Fortran news from the previous month.
+
+* [This website](#this-website)
+* [Standard Library](#standard-library)
+* [Package Manager](#package-manager)
+* [Events](#events)
+
+## This website
+
+If you came to this newsletter from elsewhere, welcome to the new Fortran website.
+We built this site mid-April and hope for it to be _the_ home of Fortran on the internet,
+which traditionally there hasn't been any to date.
+Look around and [let us know](https://github.com/fortran-lang/fortran-lang.github.io/issues)
+if you have any suggestions for improvement.
+Specifically, [Learn](/learn) and [Packages](/packages) are the pages that
+we'll be focusing on in the coming months.
+Please help us make them better!
+
+* [#90](https://github.com/fortran-lang/fortran-lang.org/pull/90)
+WIP: Mini-book on building programs
+
+## Fortran Standard Library
+
+Here's what's new in Fortran Standard Library:
+
+* [#191](https://github.com/fortran-lang/stdlib/pull/191)
+WIP: Function for computing Pearson correlations among elements of
+an array in the `stdlib_experimental_stats` module.
+
+* [#189](https://github.com/fortran-lang/stdlib/pull/189)
+WIP: Procedures for sparse matrices operations. Ongoing discussion on the API can be found
+[here](https://github.com/fortran-lang/stdlib/wiki/Stdlib-Sparse-matrix-API)
+
+* [#183](https://github.com/fortran-lang/stdlib/pull/183)
+Automatic API-doc generation and deployment of this [stdlib website](https://stdlib.fortran-lang.org)
```

Comment:
I wonder if this item is worth emphasizing more, perhaps in its own section?

Reply:
Good point. At least it should have its own paragraph ar the start of this section.